Gems/Multiplayer/Code/Source/NetworkEntity/EntityReplication/PropertyPublisher.h

```diff
@@ -27,25 +35,51 @@ namespace Multiplayer
             False,
         };
 
-        PropertyPublisher(NetEntityRole remoteNetworkRole, OwnsLifetime ownsLifetime, NetBindComponent* netBindComponent, AzNetworking::IConnection& connection);
+        PropertyPublisher(NetEntityRole remoteNetworkRole, OwnsLifetime ownsLifetime, AzNetworking::IConnection& connection);
 
+        //! Set the publishing state to "rebasing". The next record sent will be a rebase record.
```

Comment:
Again, delete is pretty obvious, but a reader may not understand what a "rebase" is. A really quick description could help quite a bit.

Reply:
This part is slim on comments because I don't feel 100% confident that I understand what a rebase is or what conditions would cause it to be set. I mostly left the rebasing piece of this alone.

Reply:
OK, after searching through the engine, there are no active code paths or tests that set rebasing. 

There's an entry point on EntityReplicationManager that is "SetEntityRebasing()", and from the code it's expected to be an Autonomous entity and it creates a full replication record minus any predictable values, and claims to be useful for client migrations. I'll repeat this information in a comment, but since it isn't used anywhere, I can't guarantee correctness.

Reply:
Looking at this closer, I _suspect_ we could kill the separate rebase state and either just use PrepareAddEntityRecord() to send everything on migration, or change that method to always subtract out predictable properties for Autonomous entities. I don't understand why migrations would want to behave differently than creations in that sense, since even "creation" doesn't line up with local creation necessarily, it's remote creation for an entity that might have been processing for a little while already.

But again, since nothing uses rebasing right now, and there's no way to test migrations currently, I'll just leave this as-is other than the couple extra comments I added.

Reply:
I double-checked against the multiserver code just to be sure, and couldn't see anything there either that relies on rebasing at the moment.